### ISSUE TITLE
can it support whisper-turbo?        

### ISSUE THREAD

**Issue by a21211** [collabora/WhisperLive#282]
can it support whisper-turbo?

**Comment by mnwato** [comment]
Yes. I have test it with [deepdml/faster-whisper-large-v3-turbo-ct2](https://huggingface.co/deepdml/faster-whisper-large-v3-turbo-ct2) and it works correctly.

**Comment by xSlikZodiac** [comment]
> Yes. I have test it with [deepdml/faster-whisper-large-v3-turbo-ct2](https://huggingface.co/deepdml/faster-whisper-large-v3-turbo-ct2) and it works correctly.

what was the performance difference? anything notable running largev3 vs standard faster whisper?

**Comment by mnwato** [comment]
Of course, it has better performance than large-v3, and the model is about half the size. However, in terms of quality, I was not satisfied with it due to a significant decrease in performance for the Persian language. (English is different and has only a slight decrease.)

**Comment by Spudra** [comment]
> Yes. I have test it with [deepdml/faster-whisper-large-v3-turbo-ct2](https://huggingface.co/deepdml/faster-whisper-large-v3-turbo-ct2) and it works correctly.

What I've noticed is that I can't use the translate function? For example with the large-v3 I can listen to the Dutch language and also transcribe it, but when I set translation to True it still gives me Dutch output. But other than that it does seem to work pretty well! Uses only half of the GPU resources compared to faster-whisper-large-v3, even though it loses a little bit of accuracy.

**Comment by makaveli10** [comment]
It does support already but with a few changes to server.py so as to add whisper-turbo as a valid model, but stay tuned for a PR which updates faster-whisper to a newer package and supports it in a future release.

**Comment by andreclerigo** [comment]
I'm a little bit confused, I ran this client and it seemed to work (I think that it is using turbo)

```python
from whisper_live.client import TranscriptionClient
client = TranscriptionClient(
  "10.0.22.16",
  9090,
  lang="en",
  translate=False,
  model="turbo",                                      # also support hf_model => `Systran/faster-whisper-small`
  use_vad=False,
  save_output_recording=False,                         # Only used for microphone input, False by Default
  output_recording_filename="./output_recording.wav", # Only used for microphone input
  max_clients=4,
  max_connection_time=600
)

client()
```

**Comment by makaveli10** [comment]
Whisper-turbo is supported in the lates version.
